suzieq/poller/controller/utils/inventory_utils.py

```diff
@@ -161,3 +180,17 @@ def get_inventory_config(conf_type: str, inventory: dict) -> dict:
                 configs[name] = conf_obj
 
     return configs
+
+
+def get_env_var(name: str, default: str = '') -> str:
+    """Read environment variable. If not present, default is returned
+
+    Args:
+        name (str): environment variable to search
+        default (str, optional): value returned if the variable doesn't exists
+        Defaults to ''.
+
+    Returns:
+        str: environment variable or default
+    """
+    return getenv(name, default)
```

Comment:
I don't see any advantages in wrapping `getenv` in `get_env_var`. Is it just to have a `default = ''`?

Reply:
It is more a styling modification. I don't like to see the import for `os.getenv` everywhere